### PR TITLE
chore: add volume check while creating invoice

### LINKED ISSUE #457: Add MAX_VOLUME check to invoice creation
Similar to the MAX_VOLUME check when sending a payment we should add the same check also when creating an invoice

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #457 

Adds volume check to invoice creation endpoints (v1 and v2)

### CODE REVIEW

File: controllers/invoice.ctrl.go [hunk 1]
```diff
@@ -24,5 +28,23 @@ func (controller *InvoiceController) Invoice(c echo.Context) error {
...
+				log.JSON{
+					"message":        "error creating invoice",
+					"error":          err,
+					"lndhub_user_id": user.ID,
+				},
+			)
+			return c.JSON(http.StatusInternalServerError, responses.GeneralServerError)
+		}
+		if volume > controller.svc.Config.MaxVolume {
+			sentry.CaptureMessage(fmt.Sprintf("transaction volume exceeded for user %d", user.ID))
```
Comment (rolznz) [inline controllers/invoice.ctrl.go:44]: should this error message include "creating invoice" so it's easier to understand where the error happened?

Comment (im-adithya) [reply controllers/invoice.ctrl.go]: The middleware should take care of this right?

Comment (rolznz) [reply controllers/invoice.ctrl.go]: Ah ok, cool!

**Review by kiwiidb — CHANGES_REQUESTED** [review]
There is another endpoint where this should be added, in addinvoice.ctrl.go

**Comment by kiwiidb** [comment]
Superseded by #462

**Pull request closed without merging**